### PR TITLE
feat: styles supporting preference pane

### PR DESCRIPTION
**Pull request by gorjan5sk**

### CODE REVIEW

File: src/css/utils/_border.scss [hunk 1]
```diff
@@ -63,3 +63,9 @@
 .inner-ring-info {
   box-shadow: inset 0 0 0 2px var(--sn-stylekit-info-color);
 }
+
+.border-bottom-1-gray {
+  border-bottom-width: 1px;
+  border-bottom-style: solid;
+  border-bottom-color: var(--sn-stylekit-border-color);
+}
```
Comment (antsgar) [inline src/css/utils/_border.scss]: For utility classes, we usually want to keep each class doing only one thing. So to achieve this same result, you'd combine the classes "border-b-1" and "border-gray-300"

**Review by antsgar — APPROVED** [review]

File: src/css/_sn.scss [hunk 2]
```diff
@@ -198,3 +198,49 @@
   @extend .border-0;
   @extend .slide-down-animation;
 }
+
+/**
+ * A button that is just an icon. Separated from .sn-button because there
+ * is almost no style overlap.
+ */
+.sn-icon-button {
+  color: var(--sn-stylekit-grey-1);
```
Comment (antsgar) [inline src/css/_sn.scss]: We should a utility class for this color and extend it here instead, so that the color class itself can be reused.

Comment (gorjan5sk) [reply src/css/_sn.scss]: Yeah, I left it like this because I couldn't find a theme consistent name for it. Any ideas?

Comment (antsgar) [reply src/css/_sn.scss]: Just color-grey-1 would be okay I think. We'll probably have to rethink the whole color scheme once all our colors are v4

File: src/css/utils/_border.scss (hunk 1 above)
Comment (gorjan5sk) [reply src/css/utils/_border.scss]: Done. Commit amended.

File: src/css/_sn.scss (hunk 2 above)
Comment (gorjan5sk) [reply src/css/_sn.scss]: Done in amended commit.

**Pull request merged**